src/dripper/DripRequestHandler.ts

```diff
@@ -20,22 +21,24 @@ const isParachainValid = (parachain: string): boolean => {
   return id > 999 && id < 10_000;
 };
 
+type HandleRequestOpts =
+  | ({ external: true; captchaResponse: string } & Omit<DripRequestType, "sender">)
+  | ({ external: false; sender: string } & Omit<DripRequestType, "captchaResponse">);
+
 export class DripRequestHandler {
-  constructor(
-    private actions: PolkadotActions,
-    private recaptcha: Recaptcha,
-  ) {}
+  // eslint-disable-next-line
```

Comment:
Are you sure this is necessary?

I just tried **on my machine** (which is not source of truth) to remove this line, and I ran:
- `yarn run fix`
- `yarn run lint`

and I didn't have any issues with the linter.

Reply:
Ok, let me try this locally again and report back.

Reply:
Would you mind pasting the output of

```
> yarn list prettier
yarn list v1.22.22
warning Filtering by arguments is deprecated. Please use the pattern option instead.
├─ @prosopo/typechain-polkadot@1.1.15
│  └─ prettier@2.8.8
├─ prettier-plugin-compactify@0.1.6
│  └─ prettier@2.8.3
└─ prettier@3.0.2
Done in 0.88s.

```

and

```
yarn list eslint
yarn list v1.22.22
warning Filtering by arguments is deprecated. Please use the pattern option instead.
├─ @prosopo/typechain-polkadot@1.1.15
│  └─ eslint@8.57.0
└─ eslint@8.47.0
Done in 0.90s.
```

? Thanks

Reply:
No matter what I do I can't get this check to succeed: https://github.com/prosopo/polkadot-testnet-faucet/actions/runs/8924631461/job/24511372506?pr=9

Reply:
> Are you sure this is necessary?
> 
> I just tried **on my machine** (which is not source of truth) to remove this line, and I ran:
> 
>     * `yarn run fix`
> 
>     * `yarn run lint`
> 
> 
> and I didn't have any issues with the linter.

Don't forget about third step:

* `yarn run fix`
* `yarn run lint`
* `yarn run format`

`lint` is `eslint` and `format` is `prettier`. They're conflicting with each other in a weird way.
If you make `format` green, `lint` goes red, and vice versa.

Reply:
As it is likely an already-existing misconfiguration of lint+prettier in the repository that allows this kind linting conflict to happen, I propose we keep the `eslint-disables` and create an issue to tackle it separately, to not block this PR.